packages/ui-extensions/docs/surfaces/point-of-sale/staticPages/pages/versions.doc.ts

```diff
@@ -64,6 +61,14 @@ Refer to the [migration guide](/docs/api/pos-ui-extensions/migrating) for more i
 - Removed in POS version: N/A
 - Release day: 1/6/2025
 
+### Important fix in POS 9.29.0
+
+- Fixed an issue where the \`FormattedTextField\` were rendering black input text in dark mode. It will now render the correct light color variant.
```

Comment:
```suggestion
- Fixed an issue where the \`FormattedTextField\` were rendering black input text in dark mode.
```

Do you think we need to explain the correct behaviour?

Reply:
were -> was, too, just noticed

Reply:
"where the \`Stack\` component was not"

so I'd add the word component for consistency with the other change below, too

Reply:
god bless you boys